src/integrations/watchers/indexable-ancestor-watcher.php

```diff
@@ -175,24 +174,6 @@ static function( $indexable ) {
 		return \array_merge( $post_indexables, $additional_indexables );
 	}
 
-	/**
-	 * Builds the hierarchy for a post.
-	 *
-	 * @param int $object_id The post id.
-	 * @param int $post_type The post type.
-	 */
-	public function build_post_hierarchy( $object_id, $post_type ) {
```

Comment:
I think theoretically we cannot move a public function to another class, in case someone has extended the `Indexable_Ancestor_Watcher` class?

Do we need a deprecation here?